pkg/trace/stats/concentrator.go

```diff
@@ -90,16 +89,14 @@ func (c *Concentrator) Run() {
 
 	log.Debug("Starting concentrator")
 
-	for i := 0; i < runtime.NumCPU(); i++ {
-		go func() {
-			for {
-				select {
-				case i := <-c.In:
-					c.addNow(i, time.Now().UnixNano())
-				}
+	go func() {
```

Comment:
Let's keep this change separate. I already did it in https://github.com/DataDog/datadog-agent/pull/5963. We should merge both.